modules/mobile/config/routes.rb

```diff
@@ -53,6 +53,7 @@
     post '/user/phones', to: 'phones#create'
     put '/user/phones', to: 'phones#update'
     delete '/user/phones', to: 'phones#destroy'
+    put '/user/preferred_name', to: 'preferred_names#update'
```

Comment:
We have this open [ticket](https://app.zenhub.com/workspaces/va-mobile---backend-61d75d03a7505500147ec7f6/issues/department-of-veterans-affairs/va-mobile-app/2937) about standardizing our routes. 

It might be worth looking into that standard to determine whether the route should be `preferred_name` or `preferred_names`. If you were to use `resources` to generate the routes, I believe it would be plural. Clearly we haven't been consistent about this but we're trying to be better about it moving forward. If the standard we're using doesn't have an opinion on this, then this is good. I'm just trying to think ahead and prevent us from having to change it later.

Reply:
The current layout actually does respect the [convention](https://guides.rubyonrails.org/routing.html#singular-resources) of a singular resource. In the case of a singular resource, the route is singularized while the controller stays plural. I'm fine if we want to break convention, but that was the reason I chose this route name

Reply:
Awesome! Thanks for the correction. Looks like we've just been often going with plural for whatever reason.